### PR TITLE
Added Assassinate to GameMode.cs

### PR DESCRIPTION
**Pull request by creepercats**

### CODE REVIEW

**Comment by BenFradet** [comment]
Looks great, could you update RiotSharp/Misc/GameModeConverter.cs also?

File: RiotSharp/Misc/GameModeConverter.cs [hunk 1]
```diff
@@ -81,6 +81,9 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 case GameMode.Siege:
                     result = "SIEGE";
                     break;
+                case GameMode.Assassinate:
+                    result = "Assassinate";
+                    break;
```
Comment (BenFradet) [inline RiotSharp/Misc/GameModeConverter.cs]: `ReadJson` needs to be updated as well

**Comment by BenFradet** [comment]
Thanks a lot merging :+1: !

**Comment by BenFradet** [comment]
Also, would you like a new release quickly (that'd be 2.5.3) or are you fine with building RiotSharp from the develop branch for a couple of weeks?

**Comment by creepercats** [comment]
Would be nice if you could release a quick update :D Thanks

**Comment by BenFradet** [comment]
OK, I'll try doing that in the following hour :+1:

**Comment by creepercats** [comment]
I appreciate it 👍

**Comment by BenFradet** [comment]
2.5.3 published 👍

**Comment by creepercats** [comment]
Thanks :D

**Comment by creepercats** [comment]
Doesint seem that the new update has the edits. https://gyazo.com/d4a5bfb5b90d02380be410643857f134
Giving me the same error as before.

**Comment by BenFradet** [comment]
mmmh weird, I'll have a look

**Pull request merged**